### PR TITLE
Create CODEOWNERS for us

### PR DESCRIPTION
**Pull request by san7890**
@EOBGames @Maurukas You two alright with this? I thought it was quite silly that we didn't have this, so I just slapped it together in a few minutes. I read the docs and I think just having our three @'s should do it for assigning reviews repo-wide.

### CODE REVIEW

**Comment by Mothblocks** [comment]
Why not just use Watch?
image

**Comment by optimumtact** [comment]
This might shock you, but you can click the watch button this repo and get notified for Every comment PR and issue made

**Comment by JohnFulpWillard** [comment]
Would this even *work*?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# We tend to forget about this place sometimes.
+
+@EOBGames @Maurukas @san7890
```
Comment (JohnFulpWillard) [inline .github/CODEOWNERS]:
```suggestion
@EOBGames @Maurukas @san7890
/Station Maps/PubbyStation/ @JohnFulpWillard 
```

**Comment by Mothblocks** [comment]
lmao, it's going to give notifications for maurukas and san whenever anyone edits the "@EOBGames" folder

**Comment by san7890** [comment]
I'm very lazy.

**Comment by san7890** [comment]
> lmao, it's going to give notifications for maurukas and san whenever anyone edits the "@EOBGames" folder

Outdated review.

**Comment by Mothblocks** [comment]
bro just lcick watch

**Comment by san7890** [comment]
oh god dammit

**Comment by optimumtact** [comment]
real

**Comment by san7890** [comment]
"code owner" has a better ring to it than "i like watching"

**Comment by san7890** [comment]
thanks i was going to have to look through documentation for like five minutes and then eventually rename every folder

**Comment by optimumtact** [comment]
it wont' work anyway because this repo doesn't run the codeowners action that allows people who dont' have commit to get notifs

**Comment by san7890** [comment]
oh right that's a thing

**Comment by san7890** [comment]
(I don't want to use the Watch feature because then we don't get per-user code owner management for the maps they have a vested interest in. Also: "Watcher" sounds pretty pervert and will probably get lost eventually. "Code Owner"... now that has a ring to it. you feel me? of course you do this is map town baby)

(and yes i totally thought of adding users to codeowners before i made this PR)

**Pull request merged**